modules/i3s/src/i3s-loader.ts

```diff
@@ -12,6 +12,7 @@ import {getUrlWithoutParams} from './lib/utils/url-utils';
 const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
 
 const TILESET_REGEX = /layers\/[0-9]+$/;
+const LOCAL_SLPK_REGEX = /\.slpk$/;
```

Comment:
Nit: SLPK doesn't necessarily need to be local, we could also use an HTTPFileSystem to load from a remote SLPK server via range requests.

Reply:
Remote `slpk` has much more concerns. Eg. it won't necesserily has `.slpk` extension. Workers are off to be able to use custom `fetch` function. That will make remote `.slpk` loading much slower then REST API. But I guess we can make a some logic for reviving the zip filesystem inside workers.